pkg/cli/profiles.go

```diff
@@ -0,0 +1,66 @@
+package cli
+
+import (
+	"github.com/cockroachdb/cockroach/pkg/clusterversion"
+	"github.com/cockroachdb/cockroach/pkg/server/autoconfig/acprovider"
+	"github.com/cockroachdb/cockroach/pkg/server/autoconfig/autoconfigpb"
+	"github.com/cockroachdb/errors"
+)
+
+type profileSetter struct {
```

Comment:
nit: in this package, `profile` is not an obvious term. I had a hard time not thinking of pprof profiles. Consider pulling this out into a package.